### ISSUE TITLE
eror 1114

### ISSUE THREAD

**Issue by sauzebn** [ValveSoftware/Dota2-Gameplay#11355]
### Description

also , the error still hasn 't passed, I can 't launch the dota in any way , although I have normal hardware, all drivers have been updated, I tried many methods , which didn 't help

**Comment by KhasaRR1** [comment]
Same, latest patch didn't fix this issue, nor it is in the patchnotes tho

**Comment by sauzebn** [comment]
> Same, latest patch didn't fix this issue, nor it is in the patchnotes tho
I agree, I can 't wait to play, but I can 't just launch the client , I 'm glad that at least I 'm not the only one
